slsim/Deflectors/DeflectorTypes/nfw_cluster.py

```diff
@@ -0,0 +1,66 @@
+from slsim.Deflectors.DeflectorTypes.deflector_base import DeflectorBase
+from slsim.Deflectors.velocity_dispersion import vel_disp_nfw_aperture
+from lenstronomy.Cosmo.lens_cosmo import LensCosmo
+
+
+class NFWCluster(DeflectorBase):
+    """Class of a NFW halo lens model with subhalos. Each subhalo is a Deflector
+    instance with its own mass and light.
+
+    required quantities in dictionary:
+    - 'halo_mass': halo mass in physical M_sol
+    - 'concentration': halo concentration
+    - 'e1_mass': eccentricity of NFW profile
+    - 'e2_mass': eccentricity of NFW profile
+    - 'z': redshift of deflector
+    """
+
+    def __init__(self, deflector_dict, subhalos_list):
```

Comment:
Please add doc string for deflector_dict and subhalos_list. Clarify what subhalos_list should contain. Might be useful to explain how halo_EPl.fits and subhaloes_table.fits can be used in this class.

Reply:
Hi @nkhadka21, I've added the docstring for that method. Also included the fits files in `test_nfw_cluster` to exemplify their use, both have an astropy table with the quantities to init the halo and subhalos respectively.